src/components/reservation/helper.ts

```diff
@@ -216,7 +216,7 @@ export const consolidatedHoldings = (branchHoldings: HoldingsV3[]) => {
 export const getInstantLoanBranchHoldings = (
   branchHoldings: HoldingsV3[],
   whitelist: AgencyBranch[],
-  instantLoanStrings: string[]
```

Comment:
I am not too keen on this change. As I see it it goes against [our decision](https://github.com/danskernesdigitalebibliotek/dpl-react/blob/develop/docs/architecture/adr-007-configuration.md) to make the mouting application provide configuration in the expected format.

If no strings are provided then we should at least be able to set a `{enabled: false}`. I guess this goes against the design in danskernesdigitalebibliotek/dpl-cms#436 where modules can register their own settings. 🤔 

If we have become wiser can we do something do update the ADR accordingly?